### PR TITLE
added screeninfo package

### PR DESCRIPTION
**Pull request by gauranshkumar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gauranshkumar** [comment]
@conda-forge-admin, please ping team

File: recipes/screeninfo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - screeninfo.enumerators
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/rr-/screeninfo
+  summary: Fetch location and size of physical screens.
+  license: NOASSERTION
```
Comment (dopplershift) [inline recipes/screeninfo/meta.yaml:41]:
```suggestion
  license: MIT AND BSD-3-Clause
```

File: recipes/screeninfo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+  run:
+    - cython  # [osx]
```
Comment (dopplershift) [inline recipes/screeninfo/meta.yaml:24]: Why is cython required at *run* time? I see it's listed in `setup.py`, but there are no `.pyx` files in the package.

Comment (gauranshkumar) [reply recipes/screeninfo/meta.yaml:24]: It's required because of the OSX AppKit Internal Dependency.

**Comment by gauranshkumar** [comment]
@dopplershift Please review the changes and do let me know if any further action is required!
Thanks for the help :P

File: recipes/screeninfo/meta.yaml (hunk 2 above)
Comment (dopplershift) [reply recipes/screeninfo/meta.yaml:24]: But you're taking care of that by depending on `pyobjc-framework-cocoa`. The `osx` enumerator in `screeninfo` just imports `AppKit`, which is provided by `pyobjc-framework-cocoa`. I don't see any line of code in `screeninfo` that directly uses cython.

Comment (gauranshkumar) [reply recipes/screeninfo/meta.yaml]: I have dug a little deeper, and actually, CPython is marked as a dependency in pip, but in reality, it's not required as so I have removed CPython ad pip check to stop the build from failing in OSX env. Hope that resolves your query.

File: recipes/screeninfo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/screeninfo-{{ version }}.tar.gz
+  sha256: 1c4bac1ca329da3f68cbc4d2fbc92256aa9bb8ff8583ee3e14f91f0a7baa69cb
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (dopplershift) [inline recipes/screeninfo/meta.yaml:19]: With Cython gone, pretty sure you don't need a compiler either.
```suggestion
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just one last clean up and I think this is good to go.

**Comment by gauranshkumar** [comment]
@dopplershift I hope everything is complete now, Thanks for the suggestion :)

**Comment by gauranshkumar** [comment]
@conda-forge-admin, please ping team

**Review by dopplershift — APPROVED** [review]

**Pull request merged**